MUICore/muicore/MUICore.proto

```diff
@@ -53,6 +54,16 @@ message CLIArguments {
     string additional_mcore_args = 21;
 }
 
+message EVMArguments {
+	string contract_path = 12;
+	string contract_name = 13;
+	string solc_bin = 14;
+	string tx_limit = 22;
+	string tx_account = 23;
+	repeated string detectors_to_exclude = 24;
```

Comment:
current implementation is that the `detectors_to_exclude` is just a list of detector names, then `additional_flags` is CLI-style